pkg/csi_driver/controller.go

```diff
@@ -862,12 +862,12 @@ func (s *controllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateSn
 			return nil, status.Error(codes.Internal, err.Error())
 		}
 	} else {
-		backupSourceCSIHandle, err := util.BackupVolumeSourceToCSIVolumeHandle(backupInfo.SourceVolumeHandle)
+		backupSourceCSIHandle, err := util.BackupVolumeSourceToCSIVolumeHandle(backupInfo.SourceInstance, backupInfo.SourceShare)
 		if err != nil {
-			return nil, status.Errorf(codes.Internal, "Cannot determine volume handle from back source %s", backupInfo.SourceVolumeHandle)
+			return nil, status.Errorf(codes.Internal, "Cannot determine volume handle from back source instance %s, share %s", backupInfo.SourceInstance, backupInfo.SourceShare)
 		}
 		if backupSourceCSIHandle != volumeID {
-			return nil, status.Errorf(codes.AlreadyExists, "Backup already exists with a different source volume %s, input source volume %s", backupInfo.SourceVolumeHandle, volumeID)
```

Comment:
Why do we choose to log the csiHandle here instead of the Source Volume Handle?

Reply:
So that we understand what exactly was compared in L869
also I removed backupInfo.SourceVolumeHandle

Reply:
Because we are comparing against `backupSourceCSIHandle` in L869.
I removed the backupInfo.SourceVolumeHandle field.

Is that your question?

Reply:
Makes sense. So the log should already have been logging `backupSourceCSIHandle` instead of `backupInfo.SourceVolumeHandle`

I just wasn't sure if it's directly related to this change, or just a drive-by cleanup.

Reply:
Ack. you are correct, logs should have already logging the variable thats being compared in the if logic